x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts

```diff
@@ -354,10 +355,11 @@ export const createSecurityRuleTypeWrapper: CreateSecurityRuleTypeWrapper =
               alertWithPersistence,
               refresh,
               ruleExecutionLogger,
-              experimentalFeatures
+              experimentalFeatures,
+              startedAtOverridden ? startedAt : undefined
```

Comment:
```suggestion
              isPreview || startedAtOverridden ? startedAt : undefined
```
Can we move `alertTimestampOverride` above this call to `bulkCreateFactory` and use the same logic for this override? It looks like this call to `bulkCreateFactory` with the override ends up setting `ALERT_START` and `ALERT_LAST_DETECTED`, and it would make sense for preview alerts to override those fields also. I think at this point, since we plan to backdate `TIMESTAMP` for backfill runs, maybe all the timestamps on preview and backfill runs could work the exact same way?

Reply:
Updated in https://github.com/elastic/kibana/pull/177640/commits/ecbbc585aa27b20d311f58f191497a5e4a7520c6